pkg/langruntime/langruntime.go

```diff
@@ -309,6 +317,8 @@ func (l *Langruntimes) GetCompilationContainer(runtime, funcName string, install
 			"cp /kubeless/*.java /kubeless/function/src/main/java/io/kubeless/ && " +
 			"cp /kubeless/function-pom.xml /kubeless/function/pom.xml 2>/dev/null || true && " +
 			"mvn package > /dev/termination-log 2>&1 && mvn install > /dev/termination-log 2>&1"
+	case strings.Contains(runtime, "julia"):
+		command = "mkdir -p /home/julia/.julia/v0.6 && cp /kubeless/REQUIRE /home/julia/.julia/v0.6 &&  julia -e 'Pkg.resolve()'"
```

Comment:
these seems to be the instructions to install dependencies not to compile the user function.